config/defaults.js

```diff
@@ -184,5 +184,13 @@ module.exports = {
     password: process.env.TESTDROID_PASSWORD
   },
 
-  dockerWorkerPrivateKey: '/etc/docker-worker-priv.pem'
+  dockerWorkerPrivateKey: '/etc/docker-worker-priv.pem',
+
+  interactive: {
+    ssl: true,
+    //time, in seconds, between end of task and end of interactive session
+    //could be shorter due to worker node shutdown
+    minTime: 10 * 60,
+    expirationAfterSession: 5 * 60
```

Comment:
Comment is wrong...

`minTime` is minimum time the container will exist.. So the minimum time you have to connect. But also time that will definitely be wasted, if people run a very short task... We might want to keep this lower, say 3 min, if people submit tasks with interactive enabled by default having it at 10min could cost a lot of waste time, right?

`expirationAfterSession` is time wasted after an interactive session... I have no problem with that being high... We rarely have interactive sessions...